rotkehlchen/db/utils.py

```diff
@@ -326,6 +326,16 @@ def table_exists(cursor: 'DBCursor', name: str, schema: str | None = None) -> bo
     return exists
 
 
+def index_exists(cursor: 'DBCursor', name: str, schema: str | None = None) -> bool:
```

Comment:
This is a test only function. So move it to tests. Otherwise it gets included in production binary for no reason